### PR TITLE
Add Ui JUnit Tests

### PR DESCRIPTION
**Pull request by nikkiDEEE**

### CODE REVIEW

File: src/test/java/seedu/clialgo/command/ExitTestModeCommandTest.java [hunk 1]
```diff
@@ -94,6 +94,7 @@ public void isRunSuccessfully_expectTrue() {
             expectedOutput ="======================================================\r\n" +
                     "Starting test mode.\r\n" +
                     "======================================================\r\n" +
+                    "ERROR HERE\r\n" +
```
Comment (ong-ck) [inline src/test/java/seedu/clialgo/command/ExitTestModeCommandTest.java:97]: Why was this added?

**Review by heejet — APPROVED** [review]
LGTM
